src/style.css

```diff
@@ -0,0 +1,77 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir.ttc);
+}
+
+* {
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+  box-sizing: border-box;
+}
+
+body {
+  margin-top: 0;
+}
+
+.search__form {
+  display: inline-block;
+  width: 100%;
+  color: #3d4e61;
+  margin: 20px auto 0;
+
+}
+
+.search__form__big {
+  line-height: 22px;
+}
+
+.search__form__small {
+  line-height: 20px;
+}
+
+.search {
+  box-sizing: border-box;
+  width: 100%;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  outline: none;
+  background-image: url("images/Search.svg");
+  background-repeat: no-repeat;
+}
+
+.search__big {
+  font-size: 16px;
+  line-height: 68px;
+  padding: 0 0 0 62px;
```

Comment:
you can reset default paddings in `search` class, and set here only that padding that you need, no need to duplicate this rule in both `search__big` and `search__small`